### PR TITLE
Add tests for root.go

### PR DESCRIPTION
**Pull request by Makpoc**

### CODE REVIEW

File: config/setup/root_test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+func TestRoot(t *testing.T) {
+
+	// Predefined error substrings
+	parseErrContent := "Parse error:"
+	unableToAccessErroContent := "Unable to access root path"
+
+	existingDirPath, err := getTempDirPath()
+	if err != nil {
+		t.Errorf("BeforeTest: Failed to find an existing directory for testing! Error was: %v", err)
```
Comment (mholt) [inline config/setup/root_test.go]: Tests should not continue if this fails, so let's do t.Fatalf here instead

**Comment by Makpoc** [comment]
Of course..  I'll fix it first time tomorrow morning.

**Comment by mholt** [comment]
Thanks for the change! I've gotta fix builds on master real quick then I'll merge this in.

**Comment by Makpoc** [comment]
It's fine with me. If you have any other functionalities in mind that are behind in coverage I would gladly start working on them as well.

**Comment by mholt** [comment]
Merging this PR since the build on master is fixed. (For the record, this PR's build is not failing because of your change.)

**Comment by mholt** [comment]
Thanks @Makpoc! Are you with us on Slack? (see the CONTRIBUTING file for the link) If so, you should coordinate with @Karthic-Hackintosh about working on tests. If not, feel free to join us there or I can suggest something here.

**Comment by Karthic-Hackintosh** [comment]
@Makpoc, please feel free to join the cadddy slack channel and interact with me on anything related to writing tests :)

**Pull request merged**